packages/nimble-components/src/rich-text-mention/mention-users/index.ts

```diff
@@ -0,0 +1,82 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
```

Comment:
The path should correspond directly to the name
rich-text-mention-users -> rich-text-mention/users/index.ts

Reply:
Updated